lib/MusicBrainz/Server/Form/Filter/Recording.pm

```diff
@@ -27,11 +33,19 @@ sub options_artist_credit_id {
     ];
 }
 
+sub options_video {
+    return [
+        { value => 1, label => l('Video only') },
+        { value => 2, label => l('Non-video only') },
```

Comment:
I feel like video should be plural, but I'm also not entirely sure.

Reply:
I do think that "Videos only" and "Non-videos only" sound better than "Video only" and "Non-video only," since it's dealing with multiple results. Similar to a sign that says "Pedestrians only" vs. "Pedestrian only."

But your alternative suggestions sound fine too.

Reply:
Oh, I see how you're looking at it - I was looking at it as in "video is a flag, so it is either video flag or no video flag". Ok, actually it makes sense to me, let's do that.